lib/sshkit/scp/command.ex

```diff
@@ -3,22 +3,29 @@ defmodule SSHKit.SCP.Command do
 
   import SSHKit.Utils
 
-  def build(mode, path, options) do
-    scp = case mode do
-      :upload -> 'scp -t'
-      :download -> 'scp -f'
-    end
+  @flags [verbose: '-v', preserve: '-p', recursive: '-r']
 
-    flags = [verbose: '-v', preserve: '-p', recursive: '-r']
+  def build(:upload, path, options) do
+    scp('-t', path, options)
```

Comment:
what does the `-t` and `-f` flag do? i can't find them in the [scp manual](http://linuxcommand.org/man_pages/scp1.html)